Test/GX3: 5059/web table crud

E-Gregorio opened the pull request:
Summary:
El Pull Request realizado incluye la implementacion del CRUD en la tabla Web y las pruebas automatizadas correspondientes, con el Framework Playwright typeScript. Se realizaron los siguientes cambios:
- Creacion del Page Objetc Model (POM), identificada con el archivo WebTablePage.ts en la carpeta test/pages/. este archivo incluye los metodos y selectores para interactuar con el SUT.
- Creacion de Archivos de pruebas GX3-5059_WebTable_CRUD.spec.ts en la carpeta tests/specs/elements/.
- Los datos de prueba estan ubicados en el Archivo FormulariosWebTables.json.

Referencias:
Historia tecnica: JIRA GX3-5059
Rama: test/GX3-5059/web-table-crud

Test Results:

- Resultados de tests en mi Area Local:
![image](https://github.com/user-attachments/assets/2c944b77-8dd0-4f6b-8b52-7e3e37833c0b)

- Resultado Ultimo Commit:
![image](https://github.com/user-attachments/assets/547e2837-fcf6-44ca-ba3c-50736695cba3)

- Resultado Pipelines CI/CD:
![image](https://github.com/user-attachments/assets/aa566638-8ed1-479a-a32f-14eb416f973f)




Reviewers:
@saiotest

Commit:
chore:(GX3:5020) Se Actualizo las dependencias.
- Se actualizo package.json con nuevas dependencias, actualizando de
manera automatica el archivo yarn.lock (Gestor de paquetes)  reflejando los cambios en el mismo.
test:(GX3:5020) Se Agrego pruebas para la funcionalidad de arrastrar y soltar
- Se Creo Archivo Page Objetc Models DragAndDropPage.ts en tests/pages/.
- Se creo Archivo test en  GX3-5020_Dragable_Drag_and_Drop.test.ts.
- Se creo Archivo utilitarios de la funcion Coordenadas Aleatorias en tests/helper/utils/.

Commit:
test: (GX3-5059) add test data for web table CRUD

Commit:
feat: (GX3-5059) Create Page Objetc Model with methodos and selectors for webTablePage.
test: (GX3-5059) Add test file to verify functionality in WebTablePage CRUD operations.

saiotest reviewed (changes requested):
Revisión en vivo con recomendaciones y correcciones! Se espera unos fixes

Commit:
refactor: (GX3-5059) improve DOM locators and rename methods in the POM
- Updated DOM locators in WebTablePage.ts for better accuracy and consistency.
- Renamed methods and functions in WebTablePage.ts to improve readability and coherence in the POM.
- Refactored test cases in GX3-5059_WebTable_CRUD.spec.ts to reflect the method name changes."
